.env

```diff
@@ -0,0 +1,8 @@
+PORT=3000
+BASE_URL=http://localhost:3000
+
+DATABASE_HOST=
+DATABASE_NAME=
+DATABASE_USER=
+DATABASE_PASSWORD=
+DATABASE_PORT=
```

Comment:
i assume everyone should create `.env.local` with proper values. Can we add that info to `README.md`?

Reply:
That's right. I will mention that in Readme file.